script.js

```diff
@@ -0,0 +1,364 @@
+initializeCalculator();
```

Comment:
There is a lot of functions in this file, making it difficult to read.

I would suggest that you separate the file into 3 or more according to [Model View Controller Design Pattern](https://developer.mozilla.org/en-US/docs/Glossary/MVC):

- Model -> containing calculator logic (only dealing with numbers),
- View -> containing all that has to do with parsing numbers, turning them into strings,
- Controller -> interface between model and view.

Here you can read about separation of concerns, cohesion, 
https://dev.to/tamerlang/separation-of-concerns-the-simple-way-4jp2

Reply:
To separate the logic into different files, you'll need to use [exports and imports of functions](https://www.makeuseof.com/how-to-import-and-export-functions-in-javascript/#:~:text=A%20function%20to%20be%20imported,file%20or%20module%20to%20load.&text=This%20will%20make%20this%20function%20available%20for%20use%20in%20our%20current%20file.) You should be able to figure this out with this document but if not, let me know please.